### PR TITLE
Embedart: remove_art_file on import

### LINKED ISSUE #1662: EmbedArt Plugin: remove_art_file doesn't seem to work
I'm running beets version 1.15. The EmbedArt plugin isn't removing the art file from the file system. 
Logfile: http://pastebin.com/n10bbdpS
Config: http://pastebin.com/ztrjd16C

### PR DESCRIPTION
**Pull request by kooimens**
Should fix #1662.

I think the fix is easy. Don't know if it's clean though. Did some tests (+/- 10 albums), all successful. This is the first time I'm using python so please let me know what I should improve:)

### CODE REVIEW

File: beetsplug/embedart.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def embed_func(lib, opts, args):
                     art.embed_album(self._log, album, maxwidth, False,
                                     compare_threshold, ifempty)
 
-                    if remove_art_file and album.artpath is not None:
+                    if remove_art_file and album.artpath:
```
Comment (kooimens) [inline beetsplug/embedart.py]: @Kraymer, I assume this line can be changed as well?

File: beetsplug/embedart.py [hunk 2]
```diff
@@ -141,3 +141,10 @@ def process_album(self, album):
             art.embed_album(self._log, album, max_width, True,
                             self.config['compare_threshold'].get(int),
                             self.config['ifempty'].get(bool))
+            if self.config['remove_art_file'] and album.artpath:
+                if os.path.isfile(album.artpath):
+                    self._log.debug(u'Removing album art file '
+                                    u'for {0}', album)
+                    os.remove(album.artpath)
+                    album.artpath = None
+                    album.store()
```
Comment (sampsyo) [inline beetsplug/embedart.py]: Could you please move this stanza into a function so that we don't need to copy & paste the same code?

**Comment by sampsyo** [comment]
Awesome! Thank you for looking into this.

A bit of refactoring will avoid code duplication. Then, we'll just need a changelog entry and we'll be ready to merge.

Thanks again! :sparkles:

File: beetsplug/embedart.py (hunk 2 above)
Comment (Kraymer) [inline beetsplug/embedart.py]: +1

Comment (Kraymer) [inline beetsplug/embedart.py]: +1

**Comment by kooimens** [comment]
You're welcome. I was really struggling with the function. I probably made some mistakes and t.b.h. I'm a bit stuck now...

**Comment by sampsyo** [comment]
You were just missing a : at the end of the function declaration. Fixing now and merging.

**Pull request merged**